tests/test_fields.py

```diff
@@ -660,7 +660,7 @@ def get_items(mapping_or_list_of_two_tuples):
     return mapping_or_list_of_two_tuples
 
 
-class FieldValues:
+class FieldValues():
```

Comment:
Why?

Reply:
Because it's a leftover from inheriting from TestCase to have @override_settings available - a problem with one test was failing because Django 5 has timezone support turned on by default. The inheritance was better and working on the test itself, after moving it, those parentheses were left and missed by me - removing when rebasing now... :D

Reply:
I meant like `ruff --select=UP039 --fix` to remove all of these:
* https://docs.astral.sh/ruff/rules/unnecessary-class-parentheses/

Reply:
I didn't know about that project, very cool if you want to learn text processing in Rust, thanks!

PS: Are you related to Santa? =P (common joke around here)